src/VisualStudio/VisualBasic/Impl/Options/IntelliSenseOptionPageControl.xaml.vb

```diff
@@ -42,8 +42,8 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
         End Sub
 
         Private Sub SetEnterKeyDefaultBehavior()
-            Dim snippetValue = Me.OptionStore.GetOption(CompletionOptionsStorage.EnterKeyBehavior, LanguageNames.VisualBasic)
-            If snippetValue = SnippetsRule.Default Then
+            Dim enterKeyRule = Me.OptionStore.GetOption(CompletionOptionsStorage.EnterKeyBehavior, LanguageNames.VisualBasic)
+            If enterKeyRule = EnterKeyRule.Default Then
```

Comment:
so in the previous code
`snippetValue` is `EnterKeyRule` but compared to a `SnippetsRule.Default`
Here change it to the correct type `EnterKeyRule`
(Complier think it's OK, emm? ) 
 But since they all default to value `0`, so no functional change in fact.